test/AutoRest.TestServer.Tests/httpInfrastructure.cs

```diff
@@ -250,45 +250,38 @@ public class HttpInfrastructureTests : TestServerTestBase
             await new HttpRetryOperations(ClientDiagnostics, pipeline, host).Head408Async());
 
         [Test]
-        [Ignore("Implement retry logic: https://github.com/Azure/autorest.csharp/issues/398")]
         public Task HttpRetry500Patch() => TestStatus(async (host, pipeline) =>
             await new HttpRetryOperations(ClientDiagnostics, pipeline, host).Patch500Async());
 
         [Test]
-        [Ignore("Implement retry logic: https://github.com/Azure/autorest.csharp/issues/398")]
         public Task HttpRetry500Put() => TestStatus(async (host, pipeline) =>
             await new HttpRetryOperations(ClientDiagnostics, pipeline, host).Put500Async());
 
         [Test]
-        [Ignore("Implement retry logic: https://github.com/Azure/autorest.csharp/issues/398")]
         public Task HttpRetry502Get() => TestStatus(async (host, pipeline) =>
             await new HttpRetryOperations(ClientDiagnostics, pipeline, host).Get502Async());
 
         [Test]
-        [Ignore("Implement retry logic: https://github.com/Azure/autorest.csharp/issues/398")]
+        [Ignore("For whatever reason, this test is returning 204 and is considered a failure. It should be returning 200 and succeed.")]
```

Comment:
It might have something to do with it being OPTIONS. But, I'd have to actually look into it. For now, I'm happy that all the other tests just worked and required no changes. Amazing.